web/policies/organization_policy.ex

```diff
@@ -8,6 +8,7 @@ defmodule CodeCorps.OrganizationPolicy do
   def create?(%User{admin: true}), do: true
   def create?(%User{admin: false}), do: false
 
+  def update?(%User{admin: true}, %Organization{}), do: true
   def update?(%User{} = user, %Organization{} = organization), do: user |> role_is_at_least_admin(organization)
```

Comment:
I guess I wondering if there was ever a chance that we passed something that wasn't a `user` for the first param or `organization` for second. If we called `update?(nil, organization)`, we would get a clause failure.